src/Component/GeneralEntity/GeneralEntityRoot/GeneralEntityRoot.tsx

```diff
@@ -271,11 +274,12 @@ export function GeneralEntityRoot<T extends BaseEntity>({
       setEditEntity(undefined);
       setId(entityId);
       form.resetFields();
+      form.setFieldsValue(defaultEntity);
```

Comment:
I assume it shouldn't be a problem, but what happens if `defaultEntity` is undefined?

Reply:
no problems identified during testing.

Reply:
no problem was found during testing.